pom.xml

```diff
@@ -69,6 +69,11 @@
       </dependency>
       <dependency>
         <groupId>io.opentracing</groupId>
+        <artifactId>opentracing-util</artifactId>
```

Comment:
hmm junk drawer? Does this artifact have things besides context propagation things inside? odd choice to not name the artifact what it contains.